kuttl-config.yml

```diff
@@ -5,10 +5,11 @@ startControlPlane: true
 crdDir: config/crd/test-resources/
 testDirs:
 - tests/e2e/
-timeout: 320
+timeout: 30
```

Comment:
Deferring to @adamrdrew or @psav if this is a reasonable timeout - each tests executes/completes for me locally far before this new value.

Reply:
Ah, good catch. I was reducing this just for local testing. I'll remove this change.